src/test/clojurescript/clara/test_rules_require.cljs

```diff
@@ -0,0 +1,63 @@
+(ns clara.test-rules-require
```

Comment:
Could you add comment in this namespace and the test-rules-data namespace explaining why these namespaces exist, including a link to the GitHub issue?  Perhaps one comment could simply refer to the other to avoid content duplication.

Reply:
Done